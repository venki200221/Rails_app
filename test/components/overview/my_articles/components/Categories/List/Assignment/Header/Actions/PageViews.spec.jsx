import React from 'react';
import { shallow } from 'enzyme';
import '../../../../../../../../../testHelper';

import PageViews from '../../../../../../../../../../app/assets/javascripts/components/overview/my_articles/components/Categories/List/Assignment/Header/Actions/PageViews';

describe('PageViews', () => {
  it('should show a link that includes details about the article', () => {
    const props = {
      article: { language: 'language', project: 'project', title: 'title' }
    };
    const component = shallow(<PageViews {...props} />);
    expect(component).toMatchSnapshot();

    const link = component.find('a');
    const href = link.props().href;
    expect(href).toContain('language');
    expect(href).toContain('project');
    expect(href).toContain('title');
  });
});
