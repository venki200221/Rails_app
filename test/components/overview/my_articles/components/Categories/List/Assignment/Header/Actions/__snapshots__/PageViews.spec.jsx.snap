// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageViews should show a link that includes details about the article 1`] = `ShallowWrapper {}`;
